src/ray/gcs/tables.h

```diff
@@ -184,6 +184,18 @@ class Log : public LogInterface<ID, Data>, virtual public PubsubInterface<ID> {
   Status CancelNotifications(const JobID &job_id, const ID &id,
                              const ClientID &client_id);
 
+  /// Delete an entire key from redis.
+  ///
+  /// \param job_id The ID of the job (= driver).
+  /// \param id The ID of the data that is deleted from the GCS.
+  void Delete(const JobID &job_id, const ID &id);
+
+  /// Delete several keys from redis.
+  ///
+  /// \param job_id The ID of the job (= driver).
+  /// \param ids The vector of IDs that are deleted from the GCS.
```

Comment:
```suggestion
  /// \param ids The vector of IDs that are deleted from the GCS.
  /// \return Void.
```

Reply:
The suggested change looks the same as the original code...

Reply:
I added `/// \return Void.`